presto-hive/src/main/java/com/facebook/presto/hive/HivePageSinkProvider.java

```diff
@@ -113,17 +114,21 @@ public HivePageSinkProvider(
     public ConnectorPageSink createPageSink(ConnectorTransactionHandle transaction, ConnectorSession session, ConnectorOutputTableHandle tableHandle, PageSinkContext pageSinkProperties)
```

Comment:
`PageSinkContext pageSinkProperties`: seems you forgot to change the param names from the type renaming